examples/tcp_client/tcp_client.cc

```diff
@@ -0,0 +1,157 @@
+/**************************************************************************/ /**
+ * @brief Linux TCP client example.
+ * @file
```

Comment:
I think it's worthwhile expanding the description a bit here, in particular highlighting this is a "minimal" implementation and is not robust to network outages and reconnects (which we should highlight are critical for users to implement in any "real" system)

Reply:
Yep, good point.